--- u/dc1394/usb/control_orig.c	2011-12-05 02:52:58.957889640 +0200
+++ u/dc1394/usb/control.c	2011-12-05 02:53:11.602501248 +0200
@@ -69,6 +69,8 @@
 } usb_products[] = {
     { 0x1e10, 0x2000 }, // Point Grey Firefly MV Color
     { 0x1e10, 0x2001 }, // Point Grey Firefly MV Mono
+    { 0x1e10, 0x2002 }, // Point Grey Firefly MV Color
+    { 0x1e10, 0x2003 }, // Point Grey Firefly MV Mono
     { 0x1e10, 0x2004 }, // Point Grey Chameleon Color
     { 0x1e10, 0x2005 }, // Point Grey Chameleon Mono
     { 0, 0 },
